src/main/java/com/marginallyclever/makelangelo/makeArt/io/LoadFilePanel.java

```diff
@@ -46,6 +44,9 @@ public LoadFilePanel(Paper paper,String filename) {
 		setLayout(new BorderLayout());
 		add(getFileSelectionPanel(filename),BorderLayout.NORTH);
 		add(mySubPanel,BorderLayout.CENTER);
+
+		fc = new FileChooser(this);
```

Comment:
It would be great if there was only one instance of FileChooser (because there is already one in the Makelangelo calss) ... to unify reopen on the last path ... but I don't see an easy way ...

Reply:
Good suggestion, I have implemented it by using `PreferencesHelper`